apps/coordinator/src/components/ScriptExplorer/UTXOSet.jsx

```diff
@@ -203,6 +242,9 @@ class UTXOSet extends React.Component {
           <TableCell>
             <Copyable text={satoshisToBitcoins(input.amountSats)} />
           </TableCell>
+          <TableCell>
+            <Chip color={color} label={label}></Chip>
```

Comment:
Just a nit, but a "cleaner" way to do this (which we don't do well in the older code of this repo unfortunately) would be as its own component:

```javascript
const DustChip = ( { amountSats, feeRate, walletConfig }) => {
    const wasteMetrics = new WasteMetrics();
    const scriptType = walletConfig.addressType;
    const config = {
      requiredSignerCount: walletConfig.quorum.requiredSigners,
      totalSignerCount: walletConfig.quorum.totalSigners,
    };
    const { lowerLimit, upperLimit } = wasteMetrics.calculateDustLimits(
      feeRate,
      scriptType,
      config,
    );
    let chipArgs = {
        color: "success",
        label: "economical",
      };

    if (amountSats <= lowerLimit) {
      chipArgs = {
        color: "error",
        label: "dust",
      };
    } else if (amountSats > lowerLimit && amountSats <= upperLimit) {
      chipArgs = {
        color: "warning",
        label: "warning",
      };
    } 

  return <Chip {...chipArgs} />
}
```

And then you can do 
```jsx
<DustChip  amountSats={input.amountSats} feeRate={feeRate} walletConfig={walletConfig} />
```

you could even probably pull the `walletConfig` from the redux state using `useSelector` and remove that as a prop that's passed around: 

```javascript
  // you can search around the codebase for this as it's used in a few places
  const walletConfig = useSelector(getWalletConfig);
```

Reply:
Updated